diff --git a/BaseTools/Conf/tools_def.template b/BaseTools/Conf/tools_def.template
index 72ca0dfa21..18788bdc06 100644
--- a/BaseTools/Conf/tools_def.template
+++ b/BaseTools/Conf/tools_def.template
@@ -86,8 +86,8 @@ DEFINE CYGWIN_BINX64           = c:/cygwin/opt/tiano/x86_64-pc-mingw64/x86_64-pc
 DEFINE GCCNOLTO_IA32_PREFIX    = ENV(GCCNOLTO_BIN)
 DEFINE GCCNOLTO_X64_PREFIX     = ENV(GCCNOLTO_BIN)
 
-DEFINE GCC_IA32_PREFIX         = ENV(GCC_BIN)
-DEFINE GCC_X64_PREFIX          = ENV(GCC_BIN)
+DEFINE GCC_IA32_PREFIX         = i386-elf-
+DEFINE GCC_X64_PREFIX          = x86_64-elf-
 DEFINE GCC_HOST_PREFIX         = ENV(GCC_HOST_BIN)
 
 DEFINE UNIX_IASL_BIN           = ENV(IASL_PREFIX)iasl
